cmd/upgrade/roles/cmd.go

```diff
@@ -668,12 +663,21 @@ func buildAccountRoleCommandsFromCluster(
 			}
 			_, err = awsClient.IsPolicyExists(policyARN)
 			hasPolicy := err == nil
+			policyAttached := false
+			if hasPolicy {
+				for _, policy := range rolePolicyDetails[accRoleName] {
+					if policy.PolicyArn == policyARN {
+						policyAttached = true
+					}
+				}
+			}
 			policyName := aws.GetPolicyName(accRoleName)
 			upgradeAccountPolicyCommands := awscbRoles.ManualCommandsForUpgradeAccountRolePolicy(
 				awscbRoles.ManualCommandsForUpgradeAccountRolePolicyInput{
 					DefaultPolicyVersion: defaultPolicyVersion,
 					RoleName:             accRoleName,
 					HasPolicy:            hasPolicy,
+					PolicyAttached:       policyAttached,
```

Comment:
Can policyAttached by renamed to something like
RequiredRolePolicyAttached?
The difference between HasPolicy and PolicyAttached isn't clear for me. :cry:

Reply:
@davidleerh 
if I change "haspolicy" to "policyExists". Are the naming of the two attributes "PolicyExists" and "PolicyAttached" more clear than current?